### ISSUE TITLE
Property "shortName" is missing on types for Event and Effect

### ISSUE THREAD

**Issue by Hubro** [zerobias/effector#165]
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%2440%20Rewarded-%237E24E3.svg" />](https://issuehunt.io/r/zerobias/effector/issues/165)

I wanted to provide a pull request for this, but I couldn't make heads or tails of where your types are defined (or generated).

**Code to reproduce the issue:**

```typescript
const RootDomain = createDomain("my-application")

RootDomain.onCreateEvent(event => {
  console.log(event.shortName)
})

RootDomain.onCreateEffect(effect => {
  console.log(effect.shortName)
})
```

**Expected behavior:**

It should work without errors.

**Actual behavior:**

> Property 'shortName' does not exist on type 'Event<unknown>'.ts(2339)

> Property 'shortName' does not exist on type 'Effect<unknown, unknown, unknown>'.ts(2339)

**Versions of packages used:**

`effector@20.1.1`

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $40.00)

- [<img src='https://avatars3.githubusercontent.com/u/44827199?v=4' alt='issuehunt' width=24 height=24> issuehunt](https://issuehunt.io/u/issuehunt) ($40.00)
### Submitted pull Requests
- [#167 Add shortName to types](https://issuehunt.io/r/zerobias/effector/pull/167)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/zerobias/effector/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

**Comment by goodmind** [comment]
https://github.com/zerobias/effector/tree/master/packages/effector

**Comment by zerobias** [comment]
Fix is ready and will be published in version 20.1.2
